roboteam_ai/src/conditions/ShotAtGoal.h

```diff
@@ -0,0 +1,27 @@
+//
+// Created by robzelluf on 3/14/19.
+//
+
+#ifndef ROBOTEAM_AI_SHOTATGOAL_H
```

Comment:
Could you maybe refactor the class to be named hasClearShot or something that is a bit less vague? ShotAtGoal also could mean that you already have shot the ball at goal

Reply:
agreed

Reply:
Done